LambdaRuntimeDockerfiles/update-dockerfile.ps1

```diff
@@ -0,0 +1,78 @@
+# This script allows to update Dockerfiles with next ASP.NET Core patch version.
+# It greps the current version from Dockerfile, increment the patch version and fetches checksum file from Microsoft server.
+# If checksum file is available, it will update the Dockerfile next version and its checksum.
+# NOTE: Microsoft only releases patch updates for runtimes, in case of a minor update the Dockerfile a manual update will be required.
```

Comment:
Can you tweak the documentation so it's very clear that this _guesses_ the next version,  It doesn't query Microsoft (ie https://versionsof.net/) for that version.